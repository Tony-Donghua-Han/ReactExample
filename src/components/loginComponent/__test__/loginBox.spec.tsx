import React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import LoginBox from "../loginBox";

describe("LoginForm", (): void => {
    let wrapper: ShallowWrapper;
    beforeEach((): void => {
        wrapper = shallow(<LoginBox />);
    });

    test("should render without crushing", (): void => {
        expect(wrapper.debug()).toMatchSnapshot();
    });
});
