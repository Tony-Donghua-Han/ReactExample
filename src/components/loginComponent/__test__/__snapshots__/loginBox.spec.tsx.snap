// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm should render without crushing 1`] = `
"<div className=\\"loginBox\\" id=\\"loginBox\\">
  <div className=\\"title\\">
    Welcome Back
  </div>
  <div className=\\"divider\\" />
  <div className=\\"signinButton\\">
    <GoogleSigninButton />
  </div>
  <div className=\\"divider\\" />
  <div className=\\"form\\">
    <Connect(LoginForm) />
  </div>
  <div className=\\"loginFooter\\">
    <div className=\\"forgetPassword link\\">
      Forget Password?
    </div>
    <div className=\\"createAccount\\">
      <div>
        Don&#39;t have an account?
      </div>
      <div className=\\"link\\">
        Get Started
      </div>
    </div>
  </div>
</div>"
`;
